tempesta_fw/addr.c

```diff
@@ -515,3 +515,4 @@ tfw_addr_sa_len(const TfwAddr *addr)
         validate_addr(addr);
         return (addr->family == AF_INET6) ? sizeof(addr->v6) : sizeof(addr->v4);
 }
+EXPORT_SYMBOL(tfw_addr_sa_len);
```

Comment:
Commented many times: do not pollute space of exported symbols names. The function is pretty short and must be moved to .h file instead of being exported.